constants.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent
+DATA = ROOT / "data"
+TRAIN_DIR = DATA / "train"
+VAL_DIR = DATA / "val"
+TEST_DIR = DATA / "test"
+
+# NO_VAL = True
+NO_VAL = False
```

Comment:
This kind of thing would be better to have handled with an `ArgumentParser` option, or alternately via an environment variable. E.g.

```suggestion
NO_VAL = os.environ.get("NO_VAL") is not None
```

Reply:
I will make the changes here when I implement a working `Argument Parser`